.changeset/stale-otters-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-button": patch
+---
+
+Update `Button.large` and `Button.small (khanmigo)` variants to use `radius_40` (4px, 0.4rem) instead of `6px`.
```

Comment:
The `(4px, 0.4rem)` is helpful here! Could you include it on the other mentions of `radius_40` so we know the new pixel equivalent on all these patch updates?